### PR TITLE
Catch Up PR

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
36 - Breadth First
 - [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

**Comment by Gr8-Dayne** [comment]
37 - Get Edge
 - [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

**Comment by Gr8-Dayne** [comment]
38 - Depth First
 - [x] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: challenges/src/main/java/graph/Edge.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    public void setUziVert(Vertex uziVert) {
+        this.uziVert = uziVert;
+    }
+
+    @Override
+    public String toString() {
+        return "Edge{" +
+                "weight=" + weight +
+                ", uziVert=" + uziVert +
```
Comment (JamesDansie) [inline challenges/src/main/java/graph/Edge.java:33]: Change this to;
                ", uziVert=" + uziVert.getName() +

File: challenges/src/main/java/graph/Vertex.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    public void setListOfEdges(List<Edge> listOfEdges) {
+        this.listOfEdges = listOfEdges;
+    }
+
+    @Override
+    public String toString() {
+        return "Vertex{" +
+                "name='" + name + '\'' +
+                ", listOfEdges=" + listOfEdges +
```
Comment (JamesDansie) [inline challenges/src/main/java/graph/Vertex.java:42]: Delete this line. Do not include the list of Edges
